audiofile/__init__.py

```diff
@@ -4,13 +4,15 @@
     channels,
     duration,
     samples,
-    sampling_rate,
 )
 from audiofile.core.io import (
     convert_to_wav,
     read,
     write,
 )
+from audiofile.core.utils import (
+    sampling_rate,
```

Comment:
This is moved from `audiofile/core/info.py` to `audiofile/core/utils.py`, which makes not really sense semantically, but otherwise we would have circular imports.

Another solution might be to make an extra `audiofile/core/sampling_rate.py` file?

Reply:
Maybe do a late important and move:

```python
from audiofile.core.info import sampling_rate as get_sampling_rate
```

in `io.py` to line 192?

Reply:
Works indeed, I changed it accordingly and `sampling_rate()` stays in `info.py`.